.github/workflows/test.yml

```diff
@@ -51,6 +51,13 @@ jobs:
           - name: Checkout Repo
             uses: actions/checkout@v3
 
+          - name: Configure AWS credentials
```

Comment:
Why do we need this?

Reply:
for s3 test

Reply:
We shouldn't use prod credentials for unit tests. Use localstack instead